tracing/tracing-zipkin/src/main/java/io/servicecomb/tracing/zipkin/ZipkinSpanAspect.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.tracing.zipkin;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Method;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import brave.Tracing;
+import io.servicecomb.tracing.Span;
+
+@Aspect
+class ZipkinSpanAspect {
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  private final ZipkinTracingAdviser adviser;
+
+  ZipkinSpanAspect(Tracing tracing) {
+    this.adviser = new ZipkinTracingAdviser(tracing.tracer());
+  }
+
+  @Around("execution(@io.servicecomb.tracing.Span * *(..)) && @annotation(spanAnnotation)")
+  public Object advise(ProceedingJoinPoint joinPoint, Span spanAnnotation) throws Throwable {
+    Method method = ((MethodSignature) joinPoint.getSignature()).getMethod();
```

Comment:
I think we can add some field in the Span to  let the user override the span name.

Reply:
yes, maybe wait for user's request, since they ask for more than just span name custimization